index.html

```diff
@@ -0,0 +1,308 @@
+<!doctype html>
+
+<html lang="en">
+	<head>
+		
+  		<title>My Portfolio</title>
+  		
+  		<link rel = "stylesheet" type = "text/css" href = "css/styles.css" />
+  		<link rel = "stylesheet" type = "text/css" href = "css/normalize.css" />
```

Comment:
swap the order around of the link tags - this is so you can override normalize with your own css